rom-hacking/scripting/animation-commands/ac-down.mdx

```diff
@@ -0,0 +1,40 @@
+# (769) _AC_DOWN
+
+## Effect
+
+Moves the actor of the animation down over a given amount of frames per tile.
+
+If the second argument is negative, 16 is used, or 8 if the player's DashFlag is set. (See EvEntityCommand$$GetWalkTime)
+
+If the third argument is provided, the entity will move while in it's idle animation.
```

Comment:
When referencing arguments in this section, you can call them by their name that you've given to them in the "Syntax" section and make them bold, instead of saying "third argument". That's what I did for the commands I added.